Graphe_chartjs/index.js

```diff
@@ -1,61 +1,69 @@
-const API_URL = "http://127.0.0.1:8000/mode_transport_univ"
+const API_URL = "http://127.0.0.1:8000/Nombre_mode_transport_univ"
 fetch(API_URL) 
    .then(res => res.json())         //recupère données format json qui sera ensuite appélé JSONdata         
    .then( JSONdata => {
 
       //lier les boutons créer dans le fichier html au fichier javascript
-      const mode_transport_univ = document.getElementById('mode_transport_univ');
-      mode_transport_univ.addEventListener('click', change_mode_transport_univ)
-   
-      const pourcent_mode_transport_univ  = document.getElementById('pourcent_mode_transport_univ');
-      pourcent_mode_transport_univ .addEventListener('click', change_pourcent_mode_transport_univ )
 
-      const nb_person_univ = document.getElementById('nb_person_univ');
-      nb_person_univ.addEventListener('click', change_nb_person_univ)
+      const person_univ_nombre = document.getElementById('person_univ_nombre');
+      person_univ_nombre.addEventListener('click', change_person_univ_nombre)
+
+      const person_univ_pourcent = document.getElementById('person_univ_pourcent');
+      person_univ_pourcent.addEventListener('click', change_person_univ_pourcent)
+
+      const mode_transport_univ_nombre = document.getElementById('mode_transport_univ_nombre');
+      mode_transport_univ_nombre.addEventListener('click', change_mode_transport_univ_nombre)
+   
+      const mode_transport_univ_pourcent  = document.getElementById('mode_transport_univ_pourcent');
+      mode_transport_univ_pourcent.addEventListener('click', change_mode_transport_univ_pourcent )
 
+      const mode_transport_univ_nombre2  = document.getElementById('mode_transport_univ_nombre2');
+      mode_transport_univ_nombre2.addEventListener('click', change_mode_transport_univ_nombre2 ) 
 
+      
+    
       const autocolors = window['chartjs-plugin-autocolors']; // génération de couleurs automatique pour les différents graphiques
       Chart.register(autocolors);  //Enregistrement de tous les graphes
       const lighten = (color, value) => Chart.helpers.color(color).lighten(value).rgbString();
+
        
-      // données creation graphique mode transport pour chaque univ
+      // données creation graphique mode transport pour chaque univ(representation1)
       const liste_univ  = JSONdata.data.map(
         function(index){  //considère tous ensemble data st des index, pour chaque index recurepère clé
           return Object.keys(index)[0]; // Object.keys(index) retourne les differents label sous forme de liste,  [0] permet de récupérer la valeur
-        })   
-      
+        }) 
```

Comment:
A la fin de l'initialisation de const liste_univ rajouter un ;

```suggestion
        }) ;
```